### ISSUE TITLE
[Videomaker] Menu close button is in a slightly different position than the menu button is. 

### ISSUE THREAD

**Issue by kjellr** [Automattic/themes#5166]
These appear to be in slightly different positions. The X button is farther up and to the right, and it looks like a bug. Can we add some CSS to correct that? 

![close-position](https://user-images.githubusercontent.com/1202812/145243765-7fdac7bb-e08e-4dae-a0ae-10e58a8559f4.gif)

**Comment by pbking** [comment]
It depends... are we OK if the Hamburger Menu is top-aligned when the view is "responsive"?

It's an easy enough change to make the responsive menu's outer border match the theme's to ensure that distance is the same.

If it's OK to adjust both of those then there's [a fix](https://github.com/Automattic/themes/pull/5145/commits/07a99b9d8f8587d7e15a6dd90bca7775311a3181) in #5145 that does exactly that.

Before:
<img src="https://user-images.githubusercontent.com/146530/145270593-934a3ffc-1fc2-4e45-a3cd-c029fe9ae88e.png" width=300>
<img src="https://user-images.githubusercontent.com/146530/145270630-ecdae189-e2cf-41b7-893c-45d092ee1626.png" width=300>


After:
<img src="https://user-images.githubusercontent.com/146530/145270433-9b74e80a-b018-439a-b046-5c174415a514.png" width=300>
<img src="https://user-images.githubusercontent.com/146530/145270461-48c591a1-f1c6-4499-8e25-fc53e12b325b.png" width=300>

**Comment by kjellr** [comment]
Hmm ok I see how this gets complicated now. Currently we can't predict the position of the menu button because it gets center-aligned vertically depending on the height of the site title, tagline. And that height can vary depending on how how many lines each of those go onto. 😅 

I don't think aligning this to the top makes sense, since it's going to be hacky to then get it lined up nicely with the site title: 

<img width="941" alt="Screen Shot 2021-12-08 at 3 22 04 PM" src="https://user-images.githubusercontent.com/1202812/145280308-a5c6ce50-14b0-41f7-a78b-43711047a7b9.png">

Instead, can we leave the navigation how it is now, but optimize the responsive menu's padding (around the ❌) so that it matches up with the positioning of the menu button under the following conditions: 

- User has _only_ the site title. 
- The site title is short, and doesn't wrap onto multiple lines. 

In other words, if the header looked like this, the menu + close buttons would line up: 

<img width="369" alt="Screen Shot 2021-12-08 at 3 35 34 PM" src="https://user-images.githubusercontent.com/1202812/145280537-17edf863-909f-4777-8b17-ed22f04a9646.png">

**Comment by scruffian** [comment]
Should this be a Gutenberg PR rather than a videomaker one?

**Comment by kjellr** [comment]
I don't think so — Gutenberg has no control over where the button is placed on the screen, so I don't think it would be able to make this work. In our case, we have a general sense of where it might be based on the theme defaults, and we're just optimizing for that.

**Comment by pbking** [comment]
> Should this be a Gutenberg PR rather than a videomaker one?

> I don't think so — Gutenberg has no control over where the button is placed on the screen, so I don't think it would be able to make this work

One thing to note: The padding around the navigation modal as well as position of the X is hard-coded.  (At least I think so) and isn't guaranteed to have the same space as the site.  I believe that this aspect SHOULD be a Gutenberg change as otherwise the alignment of those buttons couldn't happen without tweaking that.

**Comment by kjellr** [comment]
So basically: You're suggesting that themes should be able to customize the padding there w/theme.json like they can with other blocks that support padding? I agree that it makes sense, but I also think that might get complicated... maybe CSS is the right answer.

**Comment by pbking** [comment]
Yes, customized padding makes the most sense to me.  I don't see that coming until well after "responsive options" are a thing (since it's not "padding" on the "navigation block" and there isn't an "element" we can target) and I wouldn't expect that any time soon.

I think CSS is the right answer for the time being.

**Comment by scruffian** [comment]
The block could do a couple of things that it doesn't do at the moment:
- Adjust for the admin menu
- Use the site padding setting

**Comment by scruffian** [comment]
@kjellr the updated header is now on wordpress.com. How does this version look for you?

**Comment by kjellr** [comment]
Yeah, the placement looks right to me. Will close this one out. Thanks!
